### DISCUSSION TITLE
Feature Request: Integrate Fast Erase

### DISCUSSION THREAD

**Discussion by ghost** [PartialVolume/shredos.x86_64#189] [General]
I would like to propose the addition of a fast SSD erasure feature, as my current motherboard lacks support for this functionality. 

I greatly appreciate this project, and it has proven to be truly valuable.

**Comment by ggruber** [comment]
I suggest following the approaches in the last three links mentioned in https://www.ibm.com/docs/en/linux-on-systems?topic=tools-secure-data-deletion-devices

**Comment by ggruber** [comment]
(think this is basically the same as #156 )
